2d/hydrostatic_column/mpm_linearelastic.json

```diff
@@ -0,0 +1,41 @@
+{
+  "title" : "Hydrostatic Column",
+  "input_files" : {
+    "mesh" : "mesh.txt",
+    "velocity_constraints" : "velocity-constraints.txt",
+    "particles" : "particles.txt"
+  }, 
+  "mesh" : {
+    "cell_type": "ED2Q4",
```

Comment:
Does it have to be isoparametric? can we set that to false

Reply:
Yes, will add this